### ISSUE TITLE
Some valid file paths show "File does not exist." during exp build:ios

### ISSUE THREAD

**Issue by tensiondriven** [expo/expo#724]
1) make a new project with `exp init`
2) build for ios with `exp build:ios`
3) On "? Do you already have a distribution certificate you'd like us to use...", enter a longish file path like the one below:

```
? Do you already have a distribution certificate you'd like us to use,
or do you want us to manage your certificates for you? false
? Path to P12 file: /Users/j/Documents/SomePath-mobile-app/Expo/DistributionCertificate.p12
File does not exist.
```

Actual: That didn't work, so copy it to the desktop and give it a dumber name:
```
? Path to P12 file: /Users/j/Desktop/p.p12
? Certificate P12 password:
```
That worked. 

Expected: The longer valid file path should work.

**Comment by ide** [comment]
This might be a silly question but does the file at that original path exist? Could there be any upper/lowercase inconsistencies?

**Comment by tensiondriven** [comment]
Yes, the file exists.  The file path comes directly from the Finder, as I
dragged the icon from Finder into Terminal, which copies the exact file
path.

The file path does not have any spaces, which is what I thought might be
the issue at first.  It appears to be a length issue?

**Comment by thenewt15** [comment]
Also dragged my p12 file straight from finder into terminal and got the "File does not exist" error. This issue is almost a month old... is this being addressed soon?

**Comment by chillyprig** [comment]
I'm still facing the same issue. I tried it all but it always got "File does not exist". Any help would be very nice.

**Comment by Damirikys** [comment]
Same issue. "File does not exist"

**Comment by AbdallaMohamed** [comment]
@Damirikys Thank you!

**Comment by allanleonardjr** [comment]
I just ran into the same issue, "File does not exist" on `exp@49.1.0`, after dragging the file into iTerm from Finder.

After careful examination I realized that Finder is placing a space at the very end of the line. 

**Fix:** 
After dragging my file in, and deleting the space at the end, I was able to get to the next prompt `? P12 password:`

**Comment by ide** [comment]
@fson Perhaps we need to `trim()` the CLI input?

**Comment by ragozin-n** [comment]
On mac os just RMB on your .p12 file, press Option and select Copy path to file. Then Control + V in input.
